packages/@aws-cdk/aws-inspector/package.json

```diff
@@ -54,7 +54,10 @@
     "build+test+extract": "yarn build+test && yarn rosetta:extract"
   },
   "cdk-build": {
-    "cloudformation": "AWS::Inspector",
+    "cloudformation": [
+      "AWS::Inspector",
+      "AWS::InspectorV2"
```

Comment:
Ruh roh. Do we still want to do this? I think no, right?

Reply:
Not sure. Would the alternative be creating a separate `aws-inspector-v2` package? The marketing around two namespaces in CloudFormation call them `Amazon Inspector classic` and `Amazon inspector`. Would the introduction of `aws-inspector-v2` be confusing?